docutils/plan.sh

```diff
@@ -25,4 +29,20 @@ do_build() {
 
 do_install() {
   python install.py
+
+  # write our wrapper script
+  bash_path=$(hab pkg path core/bash)
+  # echo "core/bash path: ${bash_path}"
```

Comment:
I think we can remove the echo debugging 😸

Reply:
:+1:

You know, I debated... I was going to make them comments but I thought, this might be useful in the future...  but that settles it 😸